internal/internal_task_pollers.go

```diff
@@ -47,6 +48,9 @@ const (
 	stickyDecisionScheduleToStartTimeoutSeconds = 5
 
 	ratioToForceCompleteDecisionTaskComplete = 0.8
+
+	causeTag    = "Cause"
```

Comment:
might be better to put it into `internal_logging_tags.go`

Reply:
I suppose we don't have a shared metric-tag-keys definition location, do we 🤔 

looking at e.g. internal_worker.go though:
- yea, a constant here looks fine.  we can consolidate them later, but I don't see a "cause" elsewhere so I don't think this will be confused with / clobber anything.
- lowercase it like all other metric keys.

Reply:
@Groxx addressed the comment, but all the keys are camelCase in internal_logging_tags, I think they get lowercased by the MetricsScope.

Reply:
all metrics are lowercased: https://github.com/uber-go/cadence-client/blob/master/internal/common/metrics/constants.go

tbh we should probably lowercase logs too, uppercase is annoying to deal with when using a case-sensitive system.